_data/faq.yml

```diff
@@ -0,0 +1,37 @@
+---
+"@context": https://schema.org
+"@type": FAQPage
+mainEntity:
+- "@type": Question
```

Comment:
The additional `@type` field can be removed since they don't seemed to be used anywhere. Also you can simply use `question` and `answer` fields instead of creating the nested field

Reply:
Those fields are required to follow the QA schema and google search indexing needs the FAQ page in proper QA schema.

I recommend that we follow QA schema for better search visibility on google.

Reply:
I get that, indeed you are doing that in the json-ld schema. But why do we need them in the yml file ? These field values will remain same through out the schema so you don't need to put them in the yml file.

Reply:
Okay, i get your point will update the `faq.yml` file to only contain Q & A